rust/private/rustc.bzl

```diff
@@ -204,7 +206,7 @@ def get_linker_and_args(ctx, cc_toolchain, feature_configuration, rpaths):
 
     # Add linkopt's from dependencies. This includes linkopts from transitive
     # dependencies since they get merged up.
-    for dep in ctx.attr.deps:
+    for dep in getattr(attr, "deps", []):
```

Comment:
And by making `deps` separate function parameter we can make it mandatory, and we can skip gettattr.